kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/subPages/QuestionsSettings.vue

```diff
@@ -181,6 +203,15 @@
         type: Boolean,
         default: false,
       },
+      channelsFetch: {
+        type: Object,
+        required: true,
+      },
+    },
+    data() {
+      return {
+        inputMaxQuestions: 18,
```

Comment:
I think the 18 is coming from the spec in the issue, but that was just an example - this should be a dynamic value that is based on the number of available questions. So this should either be 25 (the max per section) or the `addableQuestionCount` - I don't think you need to have a new variable here.